app/assets/stylesheets/components/_index.scss

```diff
@@ -1,5 +1,6 @@
 // Import your components CSS files here.
 @import "alert";
 @import "avatar";
+@import "card";
```

Comment:
I think it's good to be a little more specific with naming.
For example "card_showpage" etc.

Reply:
As it's a component folder, I thought it would be better to keep the name general.
In my understanding, the usage of scss file for each component is global through entire application.
Maybe as for page specific style, we can create scss in pages folder.

Correct me if I'm wrong!

Reply:
Okay. I just noticed that you and Tiger had created some css files / classes with the same names, so their might be a few conflicts.